config/settings_schema.json

```diff
@@ -248,5 +248,16 @@
               "default": 400
           }
       ]
+  },
+  {
+    "label": "Product Settings",
```

Comment:
As PM mentioned in the ticket, we should add the "Skip Cart" schema setting inside the "Add to Cart" block for the single product section.